src/routes/likernft/book/store.ts

```diff
@@ -500,6 +556,27 @@ router.post(['/:classId/new', '/class/:classId/new'], jwtAuth('write:nftbook'),
     const keywords = Array.isArray(keywordString) ? keywordString : keywordString.split(',').map((k: string) => k.trim()).filter((k: string) => !!k);
     const image = metadata?.data?.metadata?.image;
 
+    let enableSignatureImage;
+    let signedMessageText;
+
+    if (signImg || memoImg) {
+      const [signSuccess, memoSuccess] = await Promise.all([
+        signImg ? uploadFile({ path: `${classId}/signature`, file: signImg }) : Promise.resolve(null),
+        memoImg ? uploadFile({ path: `${classId}/memo`, file: memoImg }) : Promise.resolve(null),
```

Comment:
```suggestion
        signImg ? uploadFile({ path: `${classId}/signature.png`, file: signImg }) : Promise.resolve(null),
        memoImg ? uploadFile({ path: `${classId}/memo.png`, file: memoImg }) : Promise.resolve(null),
```
File name in path must include file extension